index.html

```diff
@@ -37,6 +37,109 @@ <h2> Hey There. I’m Alex</h2>
+                        industry's standard</p>
+                    <li class="listbutton2">
+                        <ul><button class="button6"> html</button></ul>
+                        <ul><button class="button7"> bootstrap</button></ul>
+                        <ul><button class="button8"> Ruby</button></ul>
+                    </li>
+                    <button class="button9"> See Project</button>
+                </section>
+
+                <section class="listproject">
+                    <p class="art">Profesional Art Printing Data</p>
+                    <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the
+                        industry's standard</p>
+                    <li class="listbutton2">
+                        <ul><button class="button6"> html</button></ul>
+                        <ul><button class="button7"> bootstrap</button></ul>
+                        <ul><button class="button8"> Ruby</button></ul>
+                    </li>
+                    <button class="button9"> See Project</button>
+                </section>
+                <section class="listproject">
+                    <p class="art">Profesional Art Printing Data</p>
+                    <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the
+                        industry's standard</p>
+                    <li class="listbutton2">
+                        <ul><button class="button6"> html</button></ul>
+                        <ul><button class="button7"> bootstrap</button></ul>
+                        <ul><button class="button8"> Ruby</button></ul>
+                    </li>
+                    <button class="button9"> See Project</button>
+                </section>
+                <section class="listproject">
+                    <p class="art">Profesional Art Printing Data</p>
+                    <p class="daily"> A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been the
+                        industry's standard</p>
+                    <li class="listbutton2">
+                        <ul><button class="button6"> html</button></ul>
+                        <ul><button class="button7"> bootstrap</button></ul>
+                        <ul><button class="button8"> Ruby</button></ul>
+                    </li>
+                    <button class="button9"> See Project</button>
+                </section>
+            </div>
+            </div>
+        </section>
+        <h3>About me</h3>
+        <p class="footer">Hello I’m a software developer! I can help you build a product , feature or website Look
+            through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+            me.</p>
```

Comment:
I can feel you are under a lot of pressure to beat the deadline, but kindly take your time and provide the best work that you can. Trust me, quality will always suffice over quantity. Please make sure the about part is complete and up to standard in conformation with the figma design below. 
Figma design

![Screenshot from 2022-09-25 00-13-17](https://user-images.githubusercontent.com/65233860/192123002-7c42feaf-6e09-4fea-b8a5-0fdd3025e687.png)

Your design
![Screenshot from 2022-09-25 00-14-56](https://user-images.githubusercontent.com/65233860/192123016-06c5267c-e29c-4725-a2df-9558e118cc03.png)